database/database.go

```diff
@@ -65,9 +70,30 @@ func NewDatabaseService(dsn string) (*DatabaseService, error) {
 		}
 	}
 
-	return &DatabaseService{
-		DB: db,
-	}, nil
+	dbService := &DatabaseService{DB: db} //nolint:exhaustruct
+	err = dbService.prepareNamedQueries()
+	return dbService, err
+}
+
+func (s *DatabaseService) prepareNamedQueries() (err error) {
```

Comment:
Why separate part of the constructor to a function with side effects though
if it's a separate function, maybe it could just build one by one and return the prepared statements?

Reply:
you mean have this function return many named statements and an err at the end? not sure this would be better, this function is basically just a way to embed this into `NewDatabaseService` in a less verbose way 🤔